OpenRA.Mods.Common/Traits/SupportPowers/SupportPowerManager.cs

```diff
@@ -32,13 +32,15 @@ public class SupportPowerManager : ITick, IResolveOrder, ITechTreeElement
 		public readonly DeveloperMode DevMode;
 		public readonly TechTree TechTree;
 		public readonly Lazy<RadarPings> RadarPings;
+		public readonly Lazy<PlayerResources> PlayerResources;
 
 		public SupportPowerManager(ActorInitializer init)
 		{
 			Self = init.Self;
 			DevMode = Self.Trait<DeveloperMode>();
 			TechTree = Self.Trait<TechTree>();
 			RadarPings = Exts.Lazy(() => init.World.WorldActor.TraitOrDefault<RadarPings>());
+			PlayerResources = Exts.Lazy(() => Self.TraitOrDefault<PlayerResources>());
```

Comment:
I didn't realize that we still had old code that hadn't been ported to `INotifyCreated`. Can we please do this properly here?

Reply:
Updated.